com.github.dariasteam.cowsrevenge.json

```diff
@@ -0,0 +1,31 @@
+{
+  "app-id": "com.github.dariasteam.cowsrevenge",
+  "branch": "stable",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "1.6",
+  "sdk": "org.freedesktop.Sdk",
+  "skip-arches": ["aarch64", "arm"],
```

Comment:
ah! sorry, you need this bit in a separate file called flathub.json. This is a flathub only file rather than a generic thing for flatpak-builder.